policy/pa_test.go

```diff
@@ -401,6 +402,9 @@ func TestChallengeTypesFor(t *testing.T) {
 	t.Parallel()
 	pa := paImpl(t)
 
+	features.Set(features.Config{DNSAccount01Enabled: true})
```

Comment:
Ideally this test would run with this flag both enabled and disabled, to prove that the flag works correctly.